File: 2020/02/2.jl
function is_valid_line(line)
   m = match(r"(\d+)-(\d+) (.): (.+)", line)
   a = parse(Int32, m[1])
   b = parse(Int32, m[2])
   c = m[3][1]
   pass = m[4]

   return (pass[a] == c) ⊻ (pass[b] == c)
end

open("input") do f
    map(is_valid_line, readlines(f)) |> sum |> println
end

File: 2017/02/2.py
m = list(map(lambda l: list(map(int, l.strip().split('\t'))),
             open("input").readlines()))
s = 0
for r in m:
    for a in r:
        for b in r:
            if a > b and a % b == 0:
                s += a // b
print(s)
